components/Sphere.tsx

```diff
@@ -0,0 +1,176 @@
+import { useState, useRef, useEffect } from "react";
+import {
+  Scene,
+  PerspectiveCamera,
+  WebGLRenderer,
+  SphereGeometry,
+  MeshStandardMaterial,
+  Mesh,
+  PointLight,
+  AmbientLight,
+  Clock,
+  LoadingManager,
+  TextureLoader,
+  UniformsUtils,
+  ShaderMaterial,
+  BackSide,
+  AdditiveBlending,
+} from "three";
+import { OrbitControls } from "three/examples/jsm/controls/OrbitControls";
+import LoadingAnimation from "./LoadingAnimation";
+
+export default function Sphere() {
+  const [loaded, setLoaded] = useState(false);
+  const mountRef = useRef<HTMLDivElement>(null);
+  const sizes = {
+    width: window.innerWidth,
+    height: window.innerHeight,
+  };
+
+  useEffect(() => {
+    // Grab it here
+    const { current } = mountRef;
+    if (!current) {
+      return;
+    }
```

Comment:
Is dit nodig? (@LukasVermeire)

Reply:
Eigenlijk geen idee, stackOverflow gaf me dit als een fix